oshi-core/src/test/java/oshi/SystemInfoTest.java

```diff
@@ -118,6 +119,10 @@ public static void main(String[] args) {
         logger.info("Checking File System...");
         printFileSystem(os.getFileSystem());
 
+        logger.info("Checking OSFileStore...");
```

Comment:
You don't need to add this here.  The call above to print the File System will iterate over the OSFileStores -- the same location where you copied the information to put in the toString.  You'll end up with something like:
```
OSFileStore[] fsArray = fileSystem.getFileStores()
for (OSFileStore fs : fsArray) {
    oshi.add(fs.toString());
}
```